packages/react/src/components/OverflowMenuV2/OverflowMenuv2.stories.js

```diff
@@ -6,68 +6,63 @@
  */
 
 import React from 'react';
+import { action } from '@storybook/addon-actions';
+
 import { ArrowsVertical } from '@carbon/icons-react';
 
-import Menu from '../Menu';
-import { StoryFrame, buildMenu } from '../Menu/_storybook-utils';
+import { MenuItem, MenuItemRadioGroup, MenuItemDivider } from '../Menu';
 
-import { OverflowMenuV2 } from '.';
+import { OverflowMenuV2 } from './';
 
 export default {
-  title: 'Experimental/unstable_Menu/OverflowMenuV2',
-  component: Menu,
+  title: 'Experimental/unstable__OverflowMenuV2',
+  component: OverflowMenuV2,
 };
 
-const Story = (items, props = {}) => (
-  <StoryFrame>
-    <OverflowMenuV2 {...props}>{buildMenu(items)}</OverflowMenuV2>
-  </StoryFrame>
-);
```

Comment:
I think we still need this so that the Experimental notification shows up on the storybook, we should add it to the new Menu story as well, see https://react.carbondesignsystem.com/?path=/story/experimental-unstable-menu-overflowmenuv2--overflow-menu-v-2

Reply:
The banner stems from a time when there were almost no experimental components and I think I added it to follow the `unstable_Tree` back then which had a notification stating the component is till undergoing an a11y review before stable.

Today, I checked the other experimental components in the storybook and none had a banner or similar. Therefore I thought I'd align with the rest and remove it. Let me know if you'd rather have it added again!

Reply:
yeah we can remove for consistency, thanks for clarifying!